app/models/upload.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class Upload < ActiveRecord::Base
+  VALID_CONTENT_TYPES = [
+    "application/pdf",
+    "application/vnd.ms-excel",
+    "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+    "application/msword",
+    "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+    "image/jpeg",
+    "image/gif",
+    "image/png",
+    "text/plain"
+  ].freeze
+
+  belongs_to :user, polymorphic: true
+  has_attached_file :file
+  validates_attachment :file,
+                       presence: true,
+                       size: { in: 0..2.megabytes },
```

Comment:
Have we settled on a 2meg limit for these documents? Is it possible this will be too small for a Microsoft formatted file?